.eslintrc.json

```diff
@@ -46,7 +46,9 @@
     "no-unused-expressions": 0,
     "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
     "chai-friendly/no-unused-expressions": 2,
-    "func-names": "off"
+    "func-names": "off",
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": ["error"]
```

Comment:
Does this mean that an error is still emitted if there are unused vars? Was this so that you could disable eslint in `didStore.delete()` in this PR?

Reply:
Its not exactly related to this PR, I just noticed that eslint only warns about unused variables and allows to commit which could lead to troubles. So I decided to be more explicit when such variables are allowed